File: php/src/LinkedList/Node.php
<?php

namespace Claudiosilvajunior\Php\LinkedList;

class Node
{
    private string $element;
    private ?Node $next = null;

    public function getElement(): string
    {
        return $this->element;
    }

    public function setElement(string $element): void
    {
        $this->element = $element;
    }

    public function getNext(): ?Node
    {
        return $this?->next;
    }

    public function setNext(?Node $next): void
    {
        $this->next = $next;
    }
}

File: java/src/main/java/org/example/linkedList/Node.java
package org.example.linkedList;

public class Node {
    private String element;
    private Node next = null;

    public Node getNext() {
        return next;
    }

    public String getElement() {
        return element;
    }

    public void setElement(String element) {
        this.element = element;
    }

    public void setNext(Node next) {
        this.next = next;
    }
}
